### PR TITLE
Added a get started guide

### LINKED ISSUE #113: Possible quick start guide
I struggled for quite a while trying to get this to work. Elden ring being my first FromSoftware game, I was not familiar with the common file structure. I am willing to open a pr to add a quick start guide to the [README](https://github.com/soulsmods/ModEngine2/blob/main/README.md) if you can proof read it. 

Steps I took

1. Unzip the zip from the latest release
2. create a dir in the mod dir with the name of the mod (for example testModName)
3. edit the `eldenring_config.toml` to include the following block
```toml
mods = [
    { enabled = true, name = "default", path = "mod\\testmodeName" },
]
```

Im not sure if this is also the correct workflow for the other games that are supported, but there is also a file called `darksouls_config.toml` in the same root dir.

Edit: supporting images
afbeelding
afbeelding

### PR DESCRIPTION
**Pull request by MitchellBerend**
Resolves #113 

This pr adds a block that describes some steps to get started with the ModEngine.

I also took the liberty to add a content index and I also added Elden Ring to the supported games list

### CODE REVIEW

**Comment by MitchellBerend** [comment]
Im not sure if I should also include some images in a collapsed section, maybe the images from the issue would help make things clearer to users that prefer images?

File: README.md [hunk 1]
```diff
@@ -11,11 +18,26 @@ Mod Engine 2 is a ground up rewrite of Mod Engine, a runtime code patching and i
...
+- [x] Elden Ring
 - [ ] Dark Souls 2: SOTF
 - [ ] Dark Souls Remastered
 - [ ] Sekiro: Shadows Die Twice
 - [ ] Bloodborne
 
+## Get started guide
+
+1. Download the latest [release](https://github.com/soulsmods/ModEngine2/releases).
+2. Create a folder in the mod folder, I recommend the name of the mod for clarity.
```
Comment (garyttierney) [inline README.md:30]: Do you mean: "Create a folder in the ~~mod~~ game folder"?

**Review by garyttierney — COMMENTED** [review]
Looks ok as a quickstart, but I agree some images would add a great deal of value on top of it. I'm happy to merge as is pending that one comment.

Though, I do wonder if it'd be better to move this section up immediately after the table of contents to get it better visibility.

**Comment by MitchellBerend** [comment]
I will add some images in collapsable sections and move the guide more to the top.

There is also the downloads section that is construction. Is that supposed to link to the releases? If so, maybe that can be removed.

Ps. Im on a phone right now so I can't check easily.

**Comment by garyttierney** [comment]
> Is that supposed to link to the releases? If so, maybe that can be removed.

Yeah, agreed. That section isn't adding much to the README.

File: README.md (hunk 1 above)
Comment (MitchellBerend) [reply README.md]: No there is a mod folder included in the zip file you can download from the release section. That mod folder is what I mean here.

**Review by garyttierney — APPROVED** [review]

**Comment by garyttierney** [comment]
Looks good, thanks!

**Pull request merged**